modules/enterprise/comm/src/main/java/org/rhq/enterprise/communications/ServiceContainer.java

```diff
@@ -1246,6 +1246,15 @@ private void setupServerConnector() throws Exception {
             if (transport.equals("https")) {
                 connector_config.put("SSLImplementation", RemotingSSLImplementation.class.getName());
             }
+
+            if (transport.equals("sslservlet") || transport.equals("https")) {
+                String tomcatAuthMode = "rhq.server.tomcat.security.client-auth-mode";
+                String tomcatAuthModeValue = System.getProperty("rhq.server.tomcat.security.client-auth-mode");
+                if (!tomcatAuthModeValue.equals("true") && !tomcatAuthModeValue.equals("false")) {
```

Comment:
Should we mention that in the error message (it's not necessarily clear to anyone why ``TRUE`` is wrong and we propose ``true`` ? Otherwise looks good.